data/json/items/ammo/300BLK.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "id": "300BLK_supersonic",
+    "type": "AMMO",
+    "name": ".300 AAC Blackout",
+    "description": ".300 AAC Blackout is an intermediate cartridge that achieves ballistics similar to the 7.62x39 but allows use on the AR-15 platform.  The round is necked-up from the 5.56mm NATO, but feeds from a STANAG magazine.  It requires a specific barrel, so will not work in a standard M4 or similar carbine.  This loading is supersonic.",
+    "weight": 18,
+    "volume": 1,
+    "price": 3500,
+    "material": [ "brass", "powder" ],
+    "symbol": "=",
+    "color": "yellow",
+    "count": 30,
+    "stack_size": 47,
+    "ammo_type": "300BLK",
+    "casing": "300BLK_casing",
+    "range": 45,
+    "damage": 43,
+    "pierce": 5,
+    "dispersion": 30,
+    "recoil": 2700,
+    "effects": [ "COOKOFF" ]
+  },
+  {
+    "id": "300BLK_subsonic",
+    "copy-from": "300BLK_supersonic",
+    "type": "AMMO",
+    "name": ".300 AAC Blackout",
+    "description": ".300 AAC Blackout is an intermediate cartridge that achieves ballistics similar to the 7.62x39 but allows use on the AR-15 platform.  The round is necked-up from the 5.56mm NATO, but feeds from a STANAG magazine.  It requires a specific barrel, so will not work in a standard M4 or similar carbine.  This loading is subsonic.",
+    "weight": 21,
+    "range": 40,
+    "damage": 26,
+    "recoil": 1700
+  },
+  {
+    "id": "reloaded_300BLK",
```

Comment:
You've got a recipe for the casing but not the reloaded cartridge. For the reload recipe use a requirement type for the ammotype and remember to add uncraft recipes for your factory loads. See #31561 for example work.

Reply:
Actually, as I understand it, that IS the reloaded round. The casing recipe is in components.json. I have added an uncraft recipe, though, thx.

Reply:
You're definitely missing the reload recipe. There's no requirement for 300 BLK in data/json/requirements/ammo.json and no recipe for `reloaded_300blk` in data/json/recipes/ammo/rifle.json.